buildtools/webpack.commons.js

```diff
@@ -85,13 +85,13 @@ module.exports = function (config) {
 
   rules.push({
     test: /\.css$/,
-    use: [{loader: 'style-loader'}, {loader: 'css-loader'}],
+    use: [{loader: MiniCssExtractPlugin.loader}, {loader: 'css-loader'}],
```

Comment:
What's the advantage of this plugin ?

Reply:
Working... with it old one (managed by the community) I didn't succeed in exporting the CSS to the dist folder!

Reply:
Okay :-p